tests/src/test/java/alluxio/cli/fs/command/LsCommandIntegrationTest.java

```diff
@@ -366,4 +366,123 @@ public void lsWithFormatSpecifierCharacter() throws IOException, AlluxioExceptio
         STATE_FILE_IN_ALLUXIO, file.getPersistenceState());
     Assert.assertEquals(expected, mOutput.toString());
   }
+
+  /**
+   * Tests ls command with sort option.
+   */
+  @Test
+  @LocalAlluxioClusterResource.Config(
+          confParams = {PropertyKey.Name.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "false",
+                  PropertyKey.Name.SECURITY_AUTHENTICATION_TYPE, "NOSASL"})
+  public void lsWithSortByPath() throws IOException, AlluxioException {
+    FileSystemTestUtils
+            .createByteFile(mFileSystem, "/testRoot/testFileA", WriteType.MUST_CACHE, 50);
+    FileSystemTestUtils
+            .createByteFile(mFileSystem, "/testRoot/testFileZ", WriteType.MUST_CACHE, 10);
+    FileSystemTestUtils
+            .createByteFile(mFileSystem, "/testRoot/testLongFile", WriteType.MUST_CACHE, 100);
+    URIStatus[] files = new URIStatus[3];
+    files[0] = mFileSystem.getStatus(new AlluxioURI("/testRoot/testFileA"));
+    files[1] = mFileSystem.getStatus(new AlluxioURI("/testRoot/testFileZ"));
+    files[2] = mFileSystem.getStatus(new AlluxioURI("/testRoot/testLongFile"));
+    mFsShell.run("ls", "--sort", "path", "/testRoot");
+    String expected = "";
+    expected += getLsNoAclResultStr("/testRoot/testFileA", files[0].getCreationTimeMs(), 50,
```

Comment:
we should set the path names so that sorting by path name produces a different result from not sorting at all

Reply:
Changed the order of file creation so it it produces a different result now